config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
```

Comment:
- [ ] Hi @BasitKorede, I got a console LoadError when I try to set up your project locally on my machine. 



![Screenshot from 2022-08-27 15-32-29](https://user-images.githubusercontent.com/63087491/187034884-8bb60e13-b9c9-4774-b9f6-80deb769dcb1.png)

You are making use of two database management systems(Postgresql and sqlite3). Kindly make sure that your adapter should be set to PostgreSQL that's  `adapter: postgresql` since you are already making use of PostgreSQL in your gem file.

Reply:
Thanks for the review changes made